cf-templates/iam/config_params.json

```diff
@@ -1,5 +1,5 @@
 {
   "Parameters" : {
-    "StackPrefix": "ecs-workshop"
+    "StackPrefix": "pt-ecs-ws"
```

Comment:
@mohitkumarsharmaflux7 please change the default stack prefix with something like  `flux7-workshop-2018`